rpi-eeprom-update

```diff
@@ -468,7 +466,7 @@ lookupVersionInfo()
    fi
 
    if [ -n "${VL805_CURRENT_VERSION}" ] && [ -n "${VL805_UPDATE_VERSION}" ]; then
-      if [ "${VL805_CURRENT_VERSION}" != "${VL805_UPDATE_VERSION}" ]; then
+      if [ "${VL805_CURRENT_VERSION}" \< "${VL805_UPDATE_VERSION}" ];  then
```

Comment:
It's been a while since I've done any bash scripting, but is the backslash needed here? https://linuxize.com/post/how-to-compare-strings-in-bash/#lexicographic-comparison seems to suggest it isn't.
(and I guess there's only one space needed before the `then` :wink: )

Reply:
The article in your link is using `[[ ]]` notation for the examples and is discussing `bash` not `sh` - the latter (which is used by this script) doesn't support `[[ ]]` as these are non-POSIX `bash`-extensions. With single `[ ]` the `<` needs to be escaped to prevent it being interpreted as input redirection.

Reply:
Oh, how embarrassing - I hadn't spotted that this was using `sh` rather than `bash` :man_facepalming: